tests/functional/_intern.js

```diff
@@ -42,7 +42,17 @@ const config = {
   tunnel: "selenium",
   tunnelOptions: {
     // this tells SeleniumTunnel to download geckodriver and chromedriver
-    drivers: ["firefox", { name: "chrome", version: "2.37" }]
+    drivers: ["firefox", { name: "chrome", version: "2.38" }]
+  },
+  capabilities: {
+    "moz:firefoxOptions": {
+      args: ["-headless"],
```

Comment:
People might not like this... if they like to watch tests run in real time? Maybe I should remove it and add a commandline arg for it.

Reply:
I personally prefer the headless version. But a cmd arg sounds reasonable.

Reply:
@Regaddi maybe let's make headless the default and add a cmd arg for `--notHeadless` or something. It's only in weird debugging situations that I care to see it.